pkg/workloads/cortex/lib/log.py

```diff
@@ -28,23 +29,39 @@ def formatTime(self, record, datefmt):
         return s
 
 
-logger = logging.getLogger("cortex")
-handler = logging.StreamHandler(stream=sys.stdout)
-formatter = MyFormatter(
-    fmt="%(asctime)s:%(name)s:%(levelname)s:%(message)s", datefmt="%Y-%m-%d %H:%M:%S.%f"
-)
-handler.setFormatter(formatter)
+current_logger = None
 
-logger.addHandler(handler)
-logger.setLevel(logging.DEBUG)
+
+def register_logger(name):
+    logger = logging.getLogger(name)
+    handler = logging.StreamHandler(stream=sys.stdout)
+    formatter = MyFormatter(
+        fmt="%(asctime)s:cortex:%(levelname)s:%(message)s", datefmt="%Y-%m-%d %H:%M:%S.%f"
+    )
+    handler.setFormatter(formatter)
+
+    logger.propagate = False
+    logger.addHandler(handler)
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+
+def refresh_logger():
+    global current_logger
+    if current_logger is not None:
+        current_logger.disabled = True
```

Comment:
Does this get garbage collected? (since if the user has an `init()` function this will get called on every prediction request)

Reply:
Init() doesn't called on every prediction, only once on startup

Reply:
Yep, sorry for my confusion!